etc/workflows/ng-publish.xml

```diff
@@ -10,50 +10,73 @@
-              <label for="flagMultiQuality">Multi Quality Videos for Adaptive Streaming</label>
+              <input id="flagQuality360p" name="flagQuality360p" type="checkbox" class="configField" value="false" />
+              <label for="flagQuality360p">360p resolution</label>
             </li>
-          </ul>
-        </fieldset>
-        <fieldset>
-          <legend>Publish to:</legend>
-          <ul id="publicationConfigurationPanel" class="workflow-configuration-subpanel">
             <li>
-              <input id="publishToSearch" name="publishToSearch" type="checkbox" checked="checked" class="configField" value="true" />
-              <label for="publishToSearch">Opencast Index</label>
-              <ul id="searchConfigurationPanel" class="workflow-configuration-subpanel">
-                <li>
-                  <input id="publishToEngage" name="searchPublishType" type="radio" checked="checked" class="configField" value="true" />
-                  <label for="publishToEngage">Opencast Media Module</label>
-                </li>
-                <li>
-                  <input id="publishToAws" name="searchPublishType" type="radio" class="configField" value="true" disabled="disabled"/>
-                  <label for="publishToAws">AWS</label>
-                 </li>
-              </ul>
+              <input id="flagQuality480p" name="flagQuality480p" type="checkbox" class="configField" value="false" />
+              <label for="flagQuality480p">480p resolution</label>
             </li>
             <li>
-              <input id="publishToApi" name="publishToApi" type="checkbox" checked="checked" class="configField" value="true" />
-              <label for="publishToApi">External Applications</label>
+              <input id="flagQuality720p" name="flagQuality720p" type="checkbox" class="configField" value="true" checked="checked"/>
+              <label for="flagQuality720p">720p resolution (HDready)</label>
             </li>
             <li>
-              <input id="publishToOaiPmh" name="publishToOaiPmh" type="checkbox" checked="checked" class="configField" value="true" />
-              <label for="publishToOaiPmh">OAI-PMH Default Repository</label>
+              <input id="flagQuality1080p" name="flagQuality1080p" type="checkbox" class="configField" value="false" />
+              <label for="flagQuality1080p">1080p resolution (FullHD)</label>
             </li>
             <li>
-              <input id="publishToYouTube" name="publishToYouTube" type="checkbox" class="configField" value="true" disabled="disabled" />
-              <label for="publishToYouTube">YouTube</label>
+              <input id="flagQuality2160p" name="flagQuality2160p" type="checkbox" class="configField" value="false" />
+              <label for="flagQuality2160p">2160p resolution (UltraHD)</label>
             </li>
           </ul>
         </fieldset>
+        <fieldset>
+          <legend>Publish to:</legend>
```

Comment:
@lkiesow I'm not sure whether it might be a better idea to just omit this from the configuration channels? Otherwise, the workflow would in theory need to try to retract any publication that could exist first in case the user publishes now to less publication channels than the event has already been published.

What do you think?

Reply:
That's correct. But that also means that you cannot select where to publish anymore either to overwrite the last settings or to set them if the previous workflow did not include them. In any case, there is a slight problem, but both ways are better than what we have now :-)